index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio Setup</title>
+</head>
+<body>
+     <main>
+        <nav class="menu-icon">
+            <h4 class="logo"><a href="#">My Logo</a></h4>
+            <img class="menubars" src="menu.png" alt="Menu"> 
+        </nav>
+        <section class="homepage">
+            <div class="intro">
+                <h2>
+                    I'm Abu Talha. <br>  
+                    Glad to see <br> you!
+                </h2>                                  
+            </div>
+            <div class="detail">
+                <p>I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>    
+            </div>
+            <div class="connect">
+                <h3>LET'S CONNECT</h3>
+                <div class="logos">
+                    <ul>
+                        <li><a href=""><img src="Vector.png" alt="Twitter"></a></li>
+                        <li><a href=""><img src="Linkedin icon.png" alt="Linked in"></a></li> 
+                        <li><a href=""><img src="msg.png" alt="message"></a></li>
+                        <li><a href=""><img src="github.png" alt="github"></a></li>
+                        <li><a href=""><img src="victory.png" alt="victory"></a></li>                       
+                    </ul>
```

Comment:
- [ ] Great work on using the semantic tag `ul` `li` to align your items here, however, you may consider aligning them properly as it is on the original design. 
- [ ] Also please make sure you stick to the original design, the typo(fonts), spacing are supposed to be similar in this way your page will look great.
Here're the styles:
![image](https://github.com/AbuTalha3/Portfolio-Setup/assets/97100091/7d3638d7-6cd1-479d-888a-0564584df4c4)

Hint: You may consider using the `flexbox` property to achieve that.
### Your design :point_down: 
![image](https://github.com/AbuTalha3/Portfolio-Setup/assets/97100091/6af3cc0d-54e5-4beb-b67b-48111930c5f9)

### Figma template :point_down: 
![image](https://github.com/AbuTalha3/Portfolio-Setup/assets/97100091/a92d5ee7-4630-4098-a121-42b570d3bb05)